runtime/lua/vim/lsp/client.lua

```diff
@@ -387,6 +387,7 @@ function Client.create(config)
     capabilities = config.capabilities,
     workspace_folders = lsp._get_workspace_folders(config.workspace_folders or config.root_dir),
     root_dir = config.root_dir,
+    is_stopping = false,
     _before_init_cb = config.before_init,
```

Comment:
`is_stopped()` is already a public method, don't need another public field. Ideally we wouldn't need this field at all, but I'm assuming there is no other way.

also let's put it after `_before_init_cb` so that at least this part of the fields is sorted... 

```suggestion
    _is_stopping = false,
    _before_init_cb = config.before_init,
```